kernel/src/sched/sched_class/mod.rs

```diff
@@ -393,12 +403,28 @@ impl LocalRunQueue for PerCpuClassRqSet {
     }
 }
 
+/// Holds per-CPU load information.
+struct PerCpuLoadStats {
+    /// The number of queued threads, excluding the idle one.
+    queued_non_idle: u32,
```

Comment:
I don't think that it is desirable to let the field names revolve around the idle task, which is just a corner case that we need to take some special care. A more obvious names are show below:

```rust
struct PerCpuLoadStats {
    /// The length of the run queue (excluding the idle task).
    queue_len: u32,
    /// Is the idle task is running.
    is_idle_running: bool,
}
```